app/models/shopping_cart.rb

```diff
@@ -1,7 +0,0 @@
-class ShoppingCart < ApplicationRecord
-  acts_as_shopping_cart
-
-  def tax_pct
```

Comment:
Why remove "moms"? To put in a new feature?

Reply:
I would deal with this in a separate feature. But please do write it up in the Tracker

Reply:
The entire file is removed. The tax is in the new model.

Reply:
great, that's what we discussed in the sprint review